plugins/modules/dnf_config_manager.py

```diff
@@ -186,6 +186,7 @@ def main():
         argument_spec=module_args,
         supports_check_mode=True
     )
+    module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C', LC_CTYPE='C')
```

Comment:
This can be simplified as:
```suggestion
    module.run_command_environ_update = dict(LANGUAGE='C', LC_ALL='C')
```